string json_file
---
bool is_sucess